examples/mcts_example.py

```diff
@@ -70,7 +71,7 @@ def simple_heuristic(game: botbowl.Game, agent:botbowl.Agent):
             own += 0.1
         elif game.get_winner() is None:
             opp += 0.1
-    return 0.5 + own - opp
```

Comment:
The heuristic should return a value between 0 and 1 as if it's a prediction of the zero-sum reward at the end of the game. So I think we should keep the 0.5 here.

Reply:
Good point, thanks! It's updated!